compiler/compiler_test.go

```diff
@@ -22,8 +22,12 @@ func requireProtoEqual(t *testing.T, want, got proto.Message) {
 
 func TestFiledescriptorSet(t *testing.T) {
 	files, err := filepath.Glob("testdata/*.proto")
+	skip := map[string]bool{"testdata/17_proto2_custom_options.proto": true}
```

Comment:
Comment?

Reply:
ok. also - just realised, I forgot to save the "value" (ooops)